docs/server/commands.rst

```diff
@@ -233,7 +233,12 @@ When the :option:`--reset` option is used , all score log data is removed and
 sync_stores
 ^^^^^^^^^^^
 
-.. versionchanged:: 2.7
+.. versionchanged:: 2.9
+
+
+.. note:: Since version 2.9 all projects are managed by Pootle FS and therefore
```

Comment:
Don't we want to mark these as deprecated?

Reply:
not sure tbh - they could be useful as a collection of api calls

Reply:
Given that these commands now use Pootle FS I was wondering if we actually want to drop them or if they would stay around a bit more to provide easier way to sync projects.

So in short I would leave this as it is until we agree when to deprecate or drop the commands formally.